R/select_vpfiles.R

```diff
@@ -1,66 +1,85 @@
-#' Select vertical profiles files (\code{vp}) from computer
+#' Select vertical profile (\code{vp}) files from computer
 #'
-#' Collect a list of vp file names within a directory that comply to the given
-#' country, radar and date range combination
+#' Create a list of vertical profile (\code{vp}) files from a local directory
+#' that match a specific date and radar range. Files are selected based on their
+#' file name (not directory structure), which should be of format
+#' \code{radar_vp_yyyymmdd*.*}, such as \code{bewid_vp_20171123T1900Z_0x5.h5}.
 #'
-#' @param directory Main directory to look into recusively.
-#' @param date_min ISO format date as start of the vp file query.
-#' @param date_max ISO format date as end of the vp file query.
-#' @param country Character vector with two letter country shortcuts.
-#' @param radar Character vector with three letter radar sindicators. This
-#' can be defined independently from the countries named.
+#' @param date_min character. YYYY-MM-DD start date of file selection.
+#' @param date_max character. YYYY-MM-DD end date of file selection.
+#' @param radars character (vector). 5-letter country/radar code(s) (e.g.
+#'   \code{bejab}) of radars to include in file selection.
+#' @param directory character. Path to local directory where files should be
+#'   looked for.
 #'
-#' @return Character list of filenames that comply to the given
-#' radar/country and date range query
+#' @return Character vector of file paths that comply to the given date and
+#'   radar range.
 #'
 #' @export
-#' @importFrom lubridate as_date
 #'
 #' @examples
-#' my_directory <- "~/my/directory/"
-#' select_vpfiles(my_directory, "2016-10-01", "2017-01-31", c("be"))
-select_vpfiles <- function(directory, date_min, date_max, country = NULL,
-                           radar = NULL) {
-  if (is.null(country)) {
-    country <- "([a-z]{2})"
+#' my_path <- "~/my/directory"
+#' select_vpfiles(
+#'   date_min = "2016-10-03",
+#'   date_max = "2016-10-05",
+#'   radars = "bejab",
+#'   directory = my_path
+#' )
+select_vpfiles <- function(date_min = NULL, date_max = NULL, radars = NULL,
+                           directory = ".") {
```

Comment:
What with backward compatibility here? Could we do this similarly to previous efforts and make sure people using `radar` and `country` are either handled as well or getting a proper depcrecation warning?